api/src/test/java/uk/gov/hmcts/payment/api/validators/RemissionValidatorTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.payment.api.validators;
```

Comment:
Once Org id changes released. This test class goes unused.

Reply:
when merge happens, this file will get removed.